packages/block-editor/src/components/block-tools/use-show-block-tools.js

```diff
@@ -29,10 +29,11 @@ export function useShowBlockTools() {
 		const clientId =
 			getSelectedBlockClientId() || getFirstMultiSelectedBlockClientId();
 
-		const block = getBlock( clientId ) || { name: '', attributes: {} };
+		const block = getBlock( clientId );
```

Comment:
If I recall, we wanted to keep the structure the same if nothing was found due to memoization: `{ name: '', attributes: {} };`. I think @Mamaduka recommended this.

Reply:
It seems like the only reason for this is so that `block` can be passed to `isUnmodifiedDefaultBlock` which doesn’t guard against nullish values. Beyond that `block` is unused so it’s okay to be nullish.

Reply:
Yeah, it's just safe guard - https://github.com/WordPress/gutenberg/pull/58979#discussion_r1502978699.